README.md

```diff
@@ -227,13 +228,14 @@ If you run the CLI in a directory that does not contain a `ditto/` folder, the f
 
 The CLI outputs data from Ditto by writing files to disk in your `ditto/` directory. The format and extension of such files will correspond to the `format` property of your configuration.
 
-| **Format**  | **File Extension** |  **Common Usage**   |
-| :---------: | :----------------: | :-----------------: |
-|    flat     |        JSON        |      Web apps       |
-| structured  |        JSON        |      Web apps       |
-|     icu     |        JSON        |      Web apps       |
-|   android   |        XML         | Native Android apps |
-| ios-strings |      STRINGS       |   Native iOS apps   |
+|   **Format**    | **File Extension** |  **Common Usage**   |
+| :-------------: | :----------------: | :-----------------: |
+|      flat       |        JSON        |      Web apps       |
```

Comment:
What do you think about making these the actual file extensions? (e.g. `.json`, `.xml`, `.strings`, `.stringsdict`). At first glance I would be unsure what STRINGSDICT would generate.